flair/embeddings/base.py

```diff
@@ -15,6 +15,12 @@
 class Embeddings(torch.nn.Module):
     """Abstract base class for all embeddings. Every new type of embedding must implement these methods."""
 
+    def __init__(self):
+        """Set some attributes that would otherwise result in errors. Overwrite these in your embedding class."""
```

Comment:
You need to call the constructor of the super class here with `super().__init__()`. This should fix the unit tests.

Reply:
and I think existing attributes would have been overwritten. both should be fixed now.